Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/CsWinUiDesktopInstancing/Program.cs

```diff
@@ -65,60 +65,6 @@ public static void ReportInfo(string message)
             }
         }
 
-        private static async Task<bool> DecideRedirection()
```

Comment:
Please reorder the file so it can be correctly diffed.  It makes it very hard to differentiate between new and old code.  When I checked out your branch locally and used a more advanced diff, it shows a bunch of new unrelated code was added.